### ISSUE TITLE
VMware Working Group meeting agenda - Jul-Aug 2017

### ISSUE THREAD

**Issue by dagwieers** [ansible/community#206]
The VMware Working Group focuses on delivering modules and features to Ansible to support working with VMware. The group meets every week on Mondays at 16:00 UTC (see [calendar](https://calendar.google.com/calendar/embed?src=ansible.com_pafenslko0e2bqjgujp8f7s0do%40group.calendar.google.com&ctx=UTC)) 

More information related to the VMware Working Group is available at: https://github.com/ansible/community/tree/master/group-vmware

**Feel free to add your questions to this ticket to be discussed during the upcoming meeting.**

**Comment by dagwieers** [comment]
I would like to discuss the following PRs:
- https://github.com/ansible/ansible/pull/25140  --  vmware_cluster: PEP8, imports, cosmetics
- https://github.com/ansible/ansible/pull/25143  --  New module: vcenter_license
- https://github.com/ansible/ansible/pull/25144  --  vmware_host: Small fixes and docs updates
- https://github.com/ansible/ansible/pull/27005  -- New module: vcenter_folder

**Comment by dagwieers** [comment]
I would like to discuss the following PRs:
- https://github.com/ansible/ansible/pull/25140  --  vmware_cluster: PEP8, imports, cosmetics
- https://github.com/ansible/ansible/pull/25143  --  New module: vcenter_license
- https://github.com/ansible/ansible/pull/25144  --  vmware_host: Small fixes and docs updates
- https://github.com/ansible/ansible/pull/27005  -- New module: vcenter_folder

**Comment by jctanner** [comment]
We need to come up with a plan to deprecate vsphere_guest.
* what features is vmware_guest missing?
* should we deprecate vsphere_guest before those features are done?
* what to do with all the open issues and PRs for vsphere_guest?
* ansible-modules-core and ansible-modules-extras issues need to be cleaned out.

**Comment by DMarby** [comment]
I would like to bring up the issue of a lot of current modules not having implemented "update" functionality, only add/remove.
To name a few:
- vmware_portgroup
- vmware_vswitch
-vmware_dvswitch
- vmware_dvs_portgroup

It makes things such as modifying the default vswitch of an esxi install impossible.

I believe fixing this might be something useful to add to the roadmap, and for the future if that's not the case already require this as criteria for new modules being added.

**Comment by dagwieers** [comment]
Maybe it is time to think about what we still want to achieve for the v2.4 release. For example, it would be nice to have integration tests for all modules that will ship in v2.4. Or have [VMware coverage](https://codecov.io/gh/ansible/ansible/tree/devel/lib/ansible/modules/cloud/vmware) closer to 100% (it is now at 23.74%). Or have a chapter related to VMware management to the documentation, etc.

If we can list a few achievable goals like that, we can ask for contributors to focus on those items and divide the work like we have been doing in the Windows working group. People interested in any of these actions already can step forward and then we can plan it and track its progress an a daily and weekly basis.

**Comment by pdellaert** [comment]
The community meeting [calendar](https://calendar.google.com/calendar/embed?src=ansible.com_pafenslko0e2bqjgujp8f7s0do%40group.calendar.google.com&ctx=UTC) does not contain the VMware group meeting, can this be added?

**Comment by Akasurde** [comment]
Meeting ended Mon Jul 24 17:20:05 2017 UTC. Information about MeetBot at http://wiki.debian.org/MeetBot .
* Minutes: https://meetbot.fedoraproject.org/ansible-meeting/2017-07-24/ansible_vmware_working_group_meeting.2017-07-24-16.01.html
* Minutes (text): https://meetbot.fedoraproject.org/ansible-meeting/2017-07-24/ansible_vmware_working_group_meeting.2017-07-24-16.01.txt
* Log: https://meetbot.fedoraproject.org/ansible-meeting/2017-07-24/ansible_vmware_working_group_meeting.2017-07-24-16.01.log.html

**Comment by Akasurde** [comment]
1.  https://github.com/ansible/ansible/pull/25944 - Documentation update
1. https://github.com/ansible/ansible/pull/25857 - Support for multiple vnic in vSwitch

**Comment by pdellaert** [comment]
One more:
ansible/ansible#27236 - vmware_guest_tools

**Comment by Akasurde** [comment]
Close request: 

- https://github.com/ansible/ansible/issues/26386 - Reason - PR is merged.

**Comment by Akasurde** [comment]
Backport:

- https://github.com/ansible/ansible/pull/27464 = vmware_guest_facts
- https://github.com/ansible/ansible/pull/27456 = vmware_guest

**Comment by Akasurde** [comment]
Meeting ended Mon Jul 31 17:02:58 2017 UTC
 - Minutes: https://meetbot.fedoraproject.org/ansible-meeting/2017-07-31/ansible_vmware_working_group_meeting.2017-07-31-16.00.html
 - Minutes (text): https://meetbot.fedoraproject.org/ansible-meeting/2017-07-31/ansible_vmware_working_group_meeting.2017-07-31-16.00.txt
 - Log: https://meetbot.fedoraproject.org/ansible-meeting/2017-07-31/ansible_vmware_working_group_meeting.2017-07-31-16.00.log.html

**Comment by dagwieers** [comment]
I would like to discuss the following PRs:
- https://github.com/ansible/ansible/pull/25140  --  vmware_cluster: PEP8, imports, cosmetics
- https://github.com/ansible/ansible/pull/25143 -- vcenter_license: Manage VMware vCenter licenses *(new module)*
- https://github.com/ansible/ansible/pull/27005 -- vcenter_folder: New module to manage folders on vCenter *(new module)*

**Comment by pdellaert** [comment]
I'd like to add the following PR to that list:
* ansible/ansible#27236 -- vmware_guest_tools: New module to wait for VMware Tools to respond running/available (main use case: wait for them to come up so you can execute commands through them)

**Comment by Akasurde** [comment]
Meeting ended Mon Aug  7 18:14:43 2017 UTC
 - Minutes: https://meetbot.fedoraproject.org/ansible-meeting/2017-08-07/ansible_vmware_working_group_meeting.2017-08-07-16.00.html
 - Minutes (text): https://meetbot.fedoraproject.org/ansible-meeting/2017-08-07/ansible_vmware_working_group_meeting.2017-08-07-16.00.txt
 - Log: https://meetbot.fedoraproject.org/ansible-meeting/2017-08-07/ansible_vmware_working_group_meeting.2017-08-07-16.00.log.html

**Comment by pdellaert** [comment]
Agenda points next meeting (trivia):
* What's missing in govcsim?
* VMworld Hackathon - do we want to have a team do something useful, ideas (expand govcsim for instance)? (both US and EU, vmwa.re/hackathon, i'm at EU)

**Comment by gundalow** [comment]
**Ansible Contributor Summit 5 (Part of AnsibleFest 2017 San Francisco)**

When: 2017-09-06:
Notes: https://public.etherpad-mozilla.org/p/ansible-summit-september-2017
Event will be in person + online (Video + IRC)

Agenda is open for topics for the next Contributors Summit

Please add (and vote `+1`) for any topics.

**Comment by pdellaert** [comment]
PR to discuss: https://github.com/ansible/ansible/pull/28155

**Comment by Akasurde** [comment]
Meeting ended Mon Aug 14 17:02:20 2017 UTC

- Minutes: https://meetbot.fedoraproject.org/ansible-meeting/2017-08-14/ansible_vmware_working_group_meeting.2017-08-14-15.59.html
- Minutes (text): https://meetbot.fedoraproject.org/ansible-meeting/2017-08-14/ansible_vmware_working_group_meeting.2017-08-14-15.59.txt
- Log: https://meetbot.fedoraproject.org/ansible-meeting/2017-08-14/ansible_vmware_working_group_meeting.2017-08-14-15.59.log.html

**Comment by Akasurde** [comment]
PR to discuss : 
https://github.com/ansible/ansible/pull/28121

**Comment by gundalow** [comment]
**NOTE:** meetings are canceled for the week Aug 21st to 25th due to 'all hands' meeting for the core team. We will resume normal schedule after that.

**Comment by dagwieers** [comment]
So we have some concerns related to the state of VMware in Ansible and are collecting these concerns at: https://github.com/ansible/community/wiki/VMware:-community-plan

Anyone is welcome to add (perceived) concerns or problems on that Wiki page so we can have a discussion during [the next VMware Working Group meeting](https://github.com/ansible/community/tree/master/group-vmware) on **Monday 28/8**. We would like to focus first on the various angles, before we can discuss solutions to tackle them.

**Comment by dagwieers** [comment]
Meeting starts in about 10 minutes (16:00 UTC).

**Comment by Akasurde** [comment]
Meeting ended Mon Aug 28 17:03:00 2017 UTC

* Minutes: https://meetbot.fedoraproject.org/ansible-meeting/2017-08-28/ansible_vmware_working_group_meeting.2017-08-28-16.00.html
* Minutes (text): https://meetbot.fedoraproject.org/ansible-meeting/2017-08-28/ansible_vmware_working_group_meeting.2017-08-28-16.00.txt
 * Log: https://meetbot.fedoraproject.org/ansible-meeting/2017-08-28/ansible_vmware_working_group_meeting.2017-08-28-16.00.log.html

**Comment by gundalow** [comment]
**NOTE:** meetings are canceled for the week September 4th through 8th due to AnsibleFest San Francisco. We will resume normal schedule after that.

**Comment by Akasurde** [comment]
Please refer next [month issue page](https://github.com/ansible/community/issues/233)
